deployment/helm/resource-management-policies/balloons/templates/configmap_enable_nri.yaml

```diff
@@ -0,0 +1,36 @@
+{{- if .Values.nri.enable }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: patch-containerd-config
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "balloons-plugin.labels" . | nindent 4 }}
+data:
+  patch_nri.sh: |
+   #!/bin/bash
+
+   config_file="/home/config.toml"
+   plugin="[plugins.\"io.containerd.nri.v1.nri\"]"
+   switch="disable = false"
+   
+   cp /etc/containerd/config.toml "$config_file"
+
+   # Check if the nri entry exists in the config file
+   if grep -q "$plugin" "$config_file"; then
+     # Check if the nri is already enabled
+     if grep -q "disable = false" "$config_file"; then
```

Comment:
This looks really haphazard - some other plugin config might have a similarly nameed (`disable`) setting. Remembet that it could be a 3rd party plugin outside containerd's codebase. I'd strongly suggest to use sed here and only edit lines between ` [plugins."io.containerd.nri.v1.nri"]` and the next config setction (line starting with a bracket `[`)

Reply:
Thanks, replaced by sed now.